src/app/space/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -60,4 +68,10 @@ export class AnalyzeOverviewComponent implements OnInit, OnDestroy {
     this.selectedFlow = $event.flow;
   }
 
+  isInOtherUsersSpace(): boolean {
+    if (this.context && this.loggedInUser) {
+      return !(this.context.space.relationships['owned-by'].data.id === this.loggedInUser.id);
+    }
+    return false;
+  }
```

Comment:
I think this would be more easily readable if the function was `userOwnsSpace()` or something - ie that it returns true iff the current space is owned by the user viewing it, rather than the reverse.